server/routes/visitors.ts

```diff
@@ -113,8 +113,8 @@ export default function routes(
 
       return true
     }),
-    query('timeOfDay').custom((value: string) => (!['morning', 'afternoon'].includes(value) ? '' : value)),
-    query('dayOfTheWeek').custom((value: string) =>
+    query('timeOfDay').customSanitizer((value: string) => (!['morning', 'afternoon'].includes(value) ? '' : value)),
```

Comment:
This wasn't working as a validator (not returning true/false correctly, I think), but replaced as a sanitiser so it actually sets to defaults if any disallowed values passed.

Reply:
It was returning either an empty string or the value which is incorrect I know realise, it should return true if there is no error rather than any sort of value.